Write up: EmberJS with REST API (#771)
ember-learn/ember-blog: source/2020-09-25-the-ember-times-issue-164.md

---

> ## [EmberJS with REST API 💪](https://github.com/ember-learn/ember-blog/pull/768)

Author: ```suggestion
## [EmberJS with REST API 💪](https://jcwebconcepts.net/emberjs-with-rest-api/)
```

> [John Costanzo (@jrock2004)](https://twitter.com/jrock2004) wrote a blog titled [EmberJS with REST API](https://github.com/ember-learn/ember-blog/pull/768). The blog details about how to get your APIs to work well with Ember Data out of the box if it follows REST API conventions. If in case your APIs don't follow the REST conventions, the post details about how to modify your Ember App' serializer to make it work with Ember data. 

Author: ```suggestion
[John Costanzo (@jrock2004)](https://twitter.com/jrock2004) wrote a blog titled [EmberJS with REST API](https://jcwebconcepts.net/emberjs-with-rest-api/). The blog explains how to get your APIs to work well with Ember Data out of the box, if it follows REST API conventions. If your APIs don't follow REST conventions, the post gives details about how to modify your Ember app serializer to make it work with Ember Data. 
```

> The blog also details about using `keyForAttribute` to make some of the manual work a little easier, so you do not have to map each API properties to the ember model properties. Head over to [the blog post](https://github.com/ember-learn/ember-blog/pull/768) today and give it a read.

Author: ```suggestion
The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-data/release/classes/JSONAPISerializer/methods/keyForAttribute?anchor=keyForAttribute) to make some of the manual work a little easier, so that you do not have to map each API property to the Ember model property. Head over to [the blog post](https://jcwebconcepts.net/emberjs-with-rest-api/) today and give it a read.
```
